src/tokens.ts

```diff
@@ -131,17 +133,33 @@ export class TokenList {
     this.tokens = tokens;
   }
 
+  static async init() {
+    try {
+      const URLS = [TOKENS_JSON_LIST_URL, TEST_TOKENS_JSON_LIST_URL];
+      const fetchURL = (url: string) => axios.get(url);
+      const promises = URLS.map(fetchURL);
+      const result = await Promise.all(promises);
+      return {
+        dynamicTokens: result[0]?.data,
+        dynamicTestTokens: result[0]?.data
+      };
+    } catch (_) {
+      return {};
+    }
+  }
+
   static async create({
     env,
     host,
     snsWasmCanisterId
   }: TokenListCreateOptions = {}): Promise<TokenList> {
-    let tokensJson: JsonableTokenList = TokensJson;
+    const { dynamicTokens, dynamicTestTokens } = await this.init();
```

Comment:
I don't get it if the dynamicTokens or dynamicTestTokens may be it is an error, because it expected to be an array. We need to be smarter at handling errors here.

TS don't show the error only because it can't infer the types which returned from the init().

<img width="527" alt="image" src="https://user-images.githubusercontent.com/1233347/220129819-10c0af00-5d09-46c3-bba0-7a6b48e243d7.png">

Reply:
will work on this